gocd/templates/libs/relay.libsonnet

```diff
@@ -53,14 +53,14 @@ function(region) {
             timeout: 1200,
             elastic_profile_id: 'relay',
             tasks: [
-              gocdtasks.script(importstr '../bash/create-sentry-release.sh'),
+              gocdtasks.script(importstr '../bash/create-sentry-relay-internal-release.sh'),
```

Comment:
To double check: Is there no way to parameterize these scripts?

Reply:
I've found using environment variables are the easiest way to pass in parameters, but there may be other options using Jsonnet (i.e. string replace https://stackoverflow.com/questions/49011412/how-to-replace-string-for-each-item-in-a-list-in-jsonnet)